### ISSUE TITLE
TartanVO: A Generalizable Learning-based VO. (arXiv:2011.00359v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#197]
TartanVO: A Generalizable Learning-based VO. (arXiv:2011.00359v1 [cs.CV])<br>
https://ift.tt/361yV7J<br>
<p>We present the first learning-based visual odometry (VO) model, which generalizes to multiple datasets and real-world scenarios and outperforms geometry-based methods in challenging scenes. We achieve this by leveraging the SLAM dataset TartanAir, which provides a large amount of diverse synthetic data in challenging environments. Furthermore, to make our VO model generalize across datasets, we propose an up-to-scale loss function and incorporate the camera intrinsic parameters into the model. Experiments show that a single model, TartanVO, trained only on synthetic data, without any finetuning, can be generalized to real-world datasets such as KITTI and EuRoC, demonstrating significant advantages over the geometry-based methods on challenging trajectories. Our code is available at https://ift.tt/2TNkqi1.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
